docs/contributing/testing/naming-conventions.mdx

```diff
@@ -0,0 +1,254 @@
+# ​Naming conventions
+
+:::caution
+
+The information in this document is a work in progress. Decisions need to be discussed and feedback
+needs to be collected before any of the information in here can be considered official.
+
+:::
+
+The main goal of the name is to describe the test in a way that makes it easy to understand what the
+test does and which cases it is intending to cover. A good list of test names gives developers the
+ability to quickly review that the suite covers all important use cases, and to easily identify gaps
+when it does not.
+
+There are many ways of describing a test but what most methods have in common is that they include
+the following
+[[1]](https://dzone.com/articles/7-popular-unit-test-naming)[[2]](https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea)[[3]](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests):
```

Comment:
💭 Do these references need to be here, or just in the justification in the PR that adds this definition? I am in the latter camp as you can trace why and how a documentation change was made, but we wouldn't have to justify in the documentation itself like a paper with references.

Reply:
Removed!